.github/CODEOWNERS

```diff
@@ -79,11 +80,11 @@
 # Default image configurations to be reviewed by general CBL-Mariner devs
 /toolkit/imageconfigs/ @microsoft/cbl-mariner-devs
 
-# Package and toolchain manifests to be reviewed by general CBL-Mariner devs
-/toolkit/resources/manifests/package/pkggen_core_aarch64.txt @microsoft/cbl-mariner-devs
-/toolkit/resources/manifests/package/pkggen_core_x86_64.txt @microsoft/cbl-mariner-devs
-/toolkit/resources/manifests/package/toolchain_aarch64.txt @microsoft/cbl-mariner-devs
-/toolkit/resources/manifests/package/toolchain_x86_64.txt @microsoft/cbl-mariner-devs
+# Package and toolchain manifests to be reviewed by toolkit team
+/toolkit/resources/manifests/package/pkggen_core_aarch64.txt @microsoft/cbl-mariner-tooling
+/toolkit/resources/manifests/package/pkggen_core_x86_64.txt @microsoft/cbl-mariner-tooling
+/toolkit/resources/manifests/package/toolchain_aarch64.txt @microsoft/cbl-mariner-tooling
+/toolkit/resources/manifests/package/toolchain_x86_64.txt @microsoft/cbl-mariner-tooling
```

Comment:
I don't think tooling and toolchain teams are exactly the same thing but since you're in the tooling one and we don't have a separate toolchain team at the moment, I think this works as well.

Reply:
Yeah, I picked the closest one that already exists. Will look into what it takes to create a toolchain team

Reply:
It wasn't too hard - created a new team